students/KellyM/Lesson2/lesson 2 - series.py

```diff
@@ -0,0 +1,48 @@
+def fib(n):
+    """ This function calculates a Fibonacci Series, which is a series that
+        starts with 0 and 1, then calculates the next number by adding the
+        previous two numbers."""
+    if n == 1:
```

Comment:
This should produce the series starting with 0 index (like you did for Lucas below) so that `fib(0) == 0, fib(1) == 1` etc.